packages/client/src/client/BrowserUnsupported.tsx

```diff
@@ -1,13 +1,15 @@
 import { StrictMode } from "react"
-import ReactDOM from "react-dom"
+import { createRoot } from "react-dom/client"
 
 import { GlobalScrollbarStyle, ThemeProvider } from "@/client/theme"
 import GlobalStyle from "@/client/theme/globals"
 
 import { Modal } from "./components/Modal"
 
 export function browserUnsupported() {
-  ReactDOM.render(
+  const container = document.getElementById("root")
+  const root = createRoot(container!)
```

Comment:
React plugin reminded me that this was the old React way (not sure why that was suppressed previously (old `eslint-plugin-react`?)

Reply:
👌🏼